### ISSUE TITLE
Add Logout

### ISSUE THREAD

**Issue by hiszd** [zentered/auth0-solid-start#11]
I am trying to use this package in my development, but there are several parts of this that aren't documented here how they are meant to be used. Some of them are very important, such as how do you setup the user logging out? Anyway, some better documentation, or an example would be helpful and appreciated.
Thank you.

**Comment by PatrickHeneise** [comment]
Thanks for raising the issue. Could you be more specific about the parts that need further clarification?

Logout is actually something I haven't worked on yet (need that soon myself, been too busy with other pieces).

**Comment by hiszd** [comment]
Logging out is probably the biggest thing. Nice things like a fully fleshed out GraphQL implementation would be good too.

I setup a simple fix for logging out and can submit a PR if you want. The trouble I ran into is that I needed to logout from the server side, but the implementation you had was scoped inside of the Auth0 Provider. It also didn't work because it was returning a Response without having any request to respond to. So I setup a logout scoped outside of the Auth0 Provider and accessible from the server side and called it from an API route so there was a request that would require a response.
